source/type/stop_point.h

```diff
@@ -56,11 +57,12 @@ struct StopPoint : public Header, Nameable, hasProperties, HasMessages {
     std::vector<StopPointConnection*> stop_point_connection_list;
     std::set<Dataset*> dataset_list;
     boost::container::flat_set<Route*> route_list;
+    pbnavitia::Address address;
```

Comment:
as @azime pointed out, in principle, we should create a new struct for address and fill the `pbnavitia:Address` in pb_converter... just like `type::Admin`